src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -91,8 +91,9 @@ public ScriptFile CreateScriptFileFromFileBuffer(string filePath, string initial
         }
 
         /// <summary>
-        /// Gets an open file in the workspace.  If the file isn't open but
-        /// exists on the filesystem, load and return it.
+        /// Gets an open file in the workspace. If the file isn't open but exists on the filesystem, load and return it.
+        /// <para>IMPORTANT: Not all documents have a backing file e.g. untitled: scheme documents.  Consider using
+        /// <see cref="Workspace.TryGetFile(string, out ScriptFile)"/> instead.</para>
```

Comment:
```suggestion
        /// <see cref="Workspace.TryGetFile(string, out ScriptFile)"/> instead.</param>
```

did you mean param here too?

Reply:
No, I meant `<para>` for paragraph.  https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/para

Reply:
omg wow

then ignore these comments 😄